dracut.sh

```diff
@@ -2018,11 +2025,14 @@ if [[ $uefi = yes ]]; then
 
     [[ -s $dracutsysrootdir/usr/lib/os-release ]] && uefi_osrelease="$dracutsysrootdir/usr/lib/os-release"
     [[ -s $dracutsysrootdir/etc/os-release ]] && uefi_osrelease="$dracutsysrootdir/etc/os-release"
+    [[ -s "${dracutsysrootdir}${uefi_splash_image}" ]] && \
```

Comment:
@haraldh, do you think there needs to be a `/` between `${dracutsysroot`} and `${uefi_splash_image}` in order to accept relative paths? _Or_ should only absolute paths pass and leave the code as is?